src/components/overview/OverviewMain.jsx

```diff
@@ -41,6 +42,31 @@ class overviewMain extends React.Component {
       }
     };
     this.componentDidMount = this.componentDidMount.bind(this);
+    this.changeStyleOnClick = this.changeStyleOnClick.bind(this);
+  }
+
+  //this changes the style when you click it!
+  changeStyleOnClick(newStyleId) {
+    for (let i = 0; i < this.state.allStyles.length; i++) {
+      if (this.state.allStyles[i].style_id === newStyleId) {
+        this.setState({
+          selectedStyle: {
+            selectedStyleId: this.state.allStyles[i].style_id,
+            selectedStyleColor: this.state.allStyles[i].name,
+            selectedReducedPrice: this.state.allStyles[i].sale_price
+          }
+        });
+        this.setState({
+          images: {
+            currentImage: this.state.allStyles[i].photos[0],
+            thumbnailImages: this.state.allStyles[i].photos,
+            maximized: false,
+            zoomed: false
+          }
+        });
+        break;
```

Comment:
What is this 'break' doing? Just curious, i dont see any issue with it but I've always been told to be weary of breaks so i figured id ask.